jormungandr-lib/src/interfaces/vote.rs

```diff
@@ -471,6 +471,23 @@ mod serde_base64_bytes {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct TallyDecryptShare(#[serde(with = "serde_base64_bytes")] Vec<u8>);
```

Comment:
This structure is not used in the API. Maybe make it internal for jcli for the time being?..